### ISSUE TITLE
Example not working when deployed as war

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-web#165]
Example not working when deployed as war

```
Loading failed for the <script> with source “http://localhost:8080/xtext/xtext-ace.js”.
```

we dont have any requirejs paths set in the samples.
the path should be

```
http://localhost:8080/xtext/2.19.0-SNAPSHOT/xtext-ace.js
``` 

since we link the folder locally when starting from launcher we either should change the path there or work with fallbacks

```
				"xtext/xtext-ace": [
					"xtext/2.19.0-SNAPSHOT/xtext-ace",
					"xtext/xtext-ace"
				]
```

we might have to filter the version to avoid extensive version adoptions

**Comment by cdietrich** [comment]
this affects css and other resources too.
the wizard generates 

<link rel="stylesheet" type="text/css" href="xtext/2.19.0-SNAPSHOT/xtext-ace.css"/>

**Comment by cdietrich** [comment]
also

```
Caused by: 
java.lang.ClassNotFoundException: org.apache.log4j.Logger
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.eclipse.jetty.webapp.WebAppClassLoader.loadClass(WebAppClassLoader.java:543)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.eclipse.xtext.web.servlet.XtextServlet.<init>(XtextServlet.java:51)
```

**Comment by cdietrich** [comment]
the service url is broken too

```<noprefix>/xtext-service/update?resource=152c23c9.statemachine```

**Comment by cdietrich** [comment]
Also the loading from "server file system" does not work when deployed as war

**Comment by cdietrich** [comment]
unpackOrion is not done for war but for eclipse task only

**Comment by cdietrich** [comment]
Fixed in 2.19
